lib/rubocop/cop/rspec/context_wording.rb

```diff
@@ -65,9 +65,17 @@ class ContextWording < Base
           (block (send #rspec? { :context :shared_context } $({str dstr xstr} ...) ...) ...)
         PATTERN
 
+        # @!method allowed?(node)
+        def_node_matcher :allowed?, <<~PATTERN
+          {
+            (str #matches_allowed_pattern?)
+            ({dstr xstr} (str #matches_allowed_pattern?) ...)
```

Comment:
This reads like it accepts twoand more  arguments, and checks if it matches on the second one. Is this correct?